tests/fsharp/Compiler/Libraries/Core/Operators/CastTests.fs

```diff
@@ -2,18 +2,18 @@
 
 namespace FSharp.Compiler.UnitTests
 
-open NUnit.Framework
+open Xunit
 open FSharp.Test
 open System
 
-[<TestFixture>]
+
 module ``Cast Tests`` =
 
-    [<Test>]
+    [<Fact>]
     let ``Cast precedence over expression forms``() =
         // Regression test for FSHARP1.0:1247
         // Precedence of type annotations :> and :?> over preceeding expression forms, e.g. if-then-else etc.
 
-        Assert.IsInstanceOf<Object> (2 :> Object)
-        Assert.IsInstanceOf<Object list> [(2 :> Object)]
-        Assert.IsInstanceOf<Object list> [2 :> Object]
\ No newline at end of file
+        Assert.IsAssignableFrom<Object> (2 :> Object) |> ignore
```

Comment:
Verify if this is the correct equivalent for NUnit `IsInstanceOf`.

Reply:
This is tricky, NUnit used `Type.IsInstanceOfType`, xUnit `Assert.IsType` calls `GetType()` which returns `int` here.